geode-core/src/main/java/org/apache/geode/cache/client/internal/QueueManagerImpl.java

```diff
@@ -1423,6 +1424,11 @@ private void logError(String message, Throwable t) {
   protected class RedundancySatisfierTask extends PoolTask {
     private boolean isCancelled;
     private ScheduledFuture<?> future;
+    private QueueManagerImpl queueManager;
```

Comment:
@nonbinaryprogrammer what is the purpose of this new field? I don't think it changes the behavior of this class.

Reply:
This holds a reference to the QueueManagerImpl so that it doesn't get freed too soon. See comments above fore more detail.